### ISSUE TITLE
the fill color in circle mode was wrong 

### ISSUE THREAD

**Issue by adad184** [kirualex/KAProgressLabel#8]
should be 

``` objc

    // The Circle
    CGContextSetFillColorWithColor(context, fillColor.CGColor);
    CGContextSetLineWidth(context, _backBorderWidth);
    CGContextFillEllipseInRect(context, circleRect);
    CGContextStrokePath(context);

```

**Comment by rickyjwhong** [comment]
Agreed! Was just about to comment this

**Comment by kirualex** [comment]
Fixed in a6d525e9c6691c09f45d09091581183cfceca2d1
